db/schema.rb

```diff
@@ -33,6 +33,18 @@
     t.index ["post_id"], name: "index_likes_on_post_id"
   end
 
+  create_table "logins", force: :cascade do |t|
+    t.string "email", default: "", null: false
+    t.string "encrypted_password", default: "", null: false
+    t.string "reset_password_token"
+    t.datetime "reset_password_sent_at"
+    t.datetime "remember_created_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["email"], name: "index_logins_on_email", unique: true
+    t.index ["reset_password_token"], name: "index_logins_on_reset_password_token", unique: true
+  end
+
```

Comment:
- [ ] Your integration of Devise is incorrect. We have users table already. That is the table to which these devise columns should be added. Kindly revisit the [Getting Started section of the Devise documentation](https://github.com/heartcombo/devise#getting-started) and you will see that you should have generated Devise using the existing User model, instead of a new Login model, which has no place in our app